extension/src/externalresources/BaseAppTranslationFiles.ts

```diff
@@ -0,0 +1,49 @@
+import { readdirSync } from 'fs';
+import { resolve, basename } from 'path';
+
+import { BlobContainer } from './ExternalResources';
+
+const languageCodeJsonRE = new RegExp(/([a-z]{2}-[A-Z]{2}).json/gi);
+const sasToken = 'sv=2019-12-12&ss=f&srt=o&sp=r&se=2021-11-25T05:28:10Z&st=2020-11-24T21:28:10Z&spr=https&sig=JP3RwQVCZBo16vJCznojVIMvPOHgnDuH937ppzPmEqQ%3D';
+const baseUrl = 'https://nabaltools.file.core.windows.net/shared/base_app_lang_files/';
+export const BaseAppTranslationFiles = new BlobContainer(__dirname, baseUrl, sasToken);
+
+BaseAppTranslationFiles.addBlob('cs-cz.json');
+BaseAppTranslationFiles.addBlob('da-dk.json');
+BaseAppTranslationFiles.addBlob('de-at.json');
+BaseAppTranslationFiles.addBlob('de-ch.json');
+BaseAppTranslationFiles.addBlob('de-de.json');
+BaseAppTranslationFiles.addBlob('en-au.json');
+BaseAppTranslationFiles.addBlob('en-ca.json');
+BaseAppTranslationFiles.addBlob('en-gb.json');
+BaseAppTranslationFiles.addBlob('en-nz.json');
+BaseAppTranslationFiles.addBlob('en-us.json');
+BaseAppTranslationFiles.addBlob('es-es_tradnl.json');
```

Comment:
this one won't match /([a-z]{2}-[A-Z]{2}).json/gi
I think I saw a similar regex somewhere else outside this file as well...

Reply:
`([a-z]{2}-[a-z]{2}(_\w*)?).json` should be good enough for this case right?

Reply:
👍

Reply:
The other place you saw it was probably in *fetch-unpack-transform.py*? Which is correct. The xliff from Base App is actually named *.es-ES.xlf* but the value from attribute `target-language` is used to name the json.

Reply:
Sounds reasonable. 👍